main.js

```diff
@@ -0,0 +1,30 @@
+// Get array data from api to variable
+function fetchData(cb, func) {
+  $.ajax({
+    url: "https://randomuser.me/api/?results=15",
+    success: function (response) {
+      usersArr = response
+      cb(usersArr, func)
+    },
+    error: (xhr, text, error) => alert(text + ": Something went wrong... Please refresh the page to try again"),
+  })
+}
+
+function printData(data) {
+  console.log(data)
+}
+
+function invokeFuncWithData(data, func = 0) {
```

Comment:
`func` should not have a default value of `0` here (because `0` is not a function. You can assign `null` or not assign any default value, since if you don't pass a function through as an argument (the only case in which this default value with get assigned) the `func` parameter will evaluate to `undefined` which is falsy in any case. Same outcome.